tests/Whoops/RunTest.php

```diff
@@ -503,4 +503,33 @@ public function testSendHttpCodeWrongCode()
 
         $this->getRunInstance()->sendHttpCode(1337);
     }
+
+    /**
+     * @covers Whoops\Run::sendHttpCode
+     */
+    public function testSendExitCode()
+    {
+        $run = $this->getRunInstance();
+        $run->sendExitCode(42);
+        $this->assertEquals(42, $run->sendExitCode());
+    }
+
+    /**
+     * @covers Whoops\Run::sendExitCode
+     */
+    public function testSendExitCodeDefaultCode()
+    {
+        $run = $this->getRunInstance();
+        $this->assertEquals(1, $run->sendExitCode());
+    }
+
+    /**
+     * @covers Whoops\Run::sendExitCode
+     */
+    public function testSendExitCodeWrongCode()
+    {
+        $this->expectExceptionOfType('InvalidArgumentException');
+
+        $this->getRunInstance()->sendExitCode(255);
```

Comment:
How does this test pass? The condition allows 255 with less-or-equal operator.

Reply:
It works for me. The conditional is about when to throw, not when to allow.

Reply:
Indeed, I’ve misread it. Thanks!